migrator-tool/src/main/scala/za/co/absa/spline/migrator/ActorSystemFacade.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.migrator
+
+import akka.actor.{ActorRefFactory, ActorSystem, Terminated}
+import com.typesafe.config.ConfigFactory
+
+import scala.concurrent.Future
+
+object ActorSystemFacade {
+
+  private val akkaConf =
+    s"""akka {
+       |  actor.guardian-supervisor-strategy = "${classOf[EscalatingSupervisorStrategy].getName}"
+       |  event-handlers = ["akka.event.slf4j.Slf4jEventHandler"]
+       |}
+       |""".stripMargin
+
+  private val _actorSystem = ActorSystem("system", ConfigFactory.parseString(akkaConf))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name '_actorSystem' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4675590)